tests/functional/firefox/desktop/test_customize.py

```diff
@@ -7,7 +7,6 @@
 from pages.firefox.desktop.customize import CustomizePage
 
 
-@pytest.mark.sanity
```

Comment:
Not directly related to this PR, but I'm removing this marker as I noticed the test should not really be marked as `sanity`.